src/components/CodeEditor.tsx

```diff
@@ -91,7 +91,7 @@ export default function CodeEditor(props: CodeEditorProps) {
             }
           }
         });
-
+        editor.setSize('auto', '100vh');
```

Comment:
The height of the code editor should be calculated by subtracting the height of the NavBar.

```suggestion
        // We need to subtract the height of NavBar.
        editor.setSize('auto', 'calc(100vh - 64px)');
```

https://forum.webflow.com/t/viewport-height-needs-to-be-100-minus-60px/70581/2